src/jobs/models/jobManager.ts

```diff
@@ -77,6 +97,15 @@ export class JobManager {
     await this.transactionManager.resetJob(jobId, newExpirationDate);
   }
 
+  private async getAvailableActions(job: IGetJobResponse): Promise<IAvailableActions> {
+    const availableActions: IAvailableActions = {
+      isResumable: (await this.isResettable({ jobId: job.id })).isResettable,
```

Comment:
separate "await this.isResettable({ jobId: job.id })" to a different line

Reply:
done